src/rastervision/core/predict_package_test.py

```diff
@@ -134,6 +134,39 @@ def test_channel_order(self):
             list(scene.raster_source.raster_transformer.channel_order),
             channel_order)
 
+    def test_ingores_no_ground_truth(self):
+        print(os.path.curdir)
```

Comment:
Reminder to remove print statements. Also `test_ingores` should be `test_ignores`

Reply:
Thanks for the catch, fixed.